BankApp/BankLibrary/Account.cs

```diff
@@ -0,0 +1,84 @@
+using System;
+
+namespace BankLibrary
+{
+    public delegate void AccountNotify(string message);
+    public abstract class Account
+    {
+        public event AccountNotify Created;
+        public event AccountNotify Withdraw;
+        public event AccountNotify Close;
+        public event AccountNotify Put;
+        public event AccountNotify Info;
+        public event AccountNotify Calculate;
+        private decimal _amount;
+        private int _id;
+        private static int _counter = 0;
+        private int _days = 0;
+        private AccountState _state;
+        public AccountState State => _state;
+        public int Days => _days;
+        public int Id => _id;
+        public abstract AccountType Type { get; }
+        public void IncrementDays() => _days++;
+        public Account(decimal amount)
+        {
+            _amount = amount;
+            _state = AccountState.Created;
+            _id = ++_counter;
+        }
+
+        public virtual void CalculatePercentage(decimal procent)
+        {
+            decimal result = _amount / 100 * procent;
```

Comment:
```suggestion
            decimal result = _amount / 100 * percent;
```
try to install spell check extension
https://marketplace.visualstudio.com/items?itemName=EWoodruff.VisualStudioSpellCheckerVS2017andLater